docs/articles/multi-attribute-semantic-search.md

```diff
@@ -418,12 +426,27 @@ weights = {
-| Harmonic Coral | 0.19245  | Branching, musical instrument-like structure with vibrating tendrils | Shallow seas and tidal pools                        | Creates complex melodies to communicate and influence emotions |
+| :------------- | -------: | :------------------------------------------------------------------- | :-------------------------------------------------- | :------------------------------------------------------------- |
+| Harmonic Coral |  0.19245 | Branching, musical instrument-like structure with vibrating tendrils | Shallow seas and tidal pools                        | Creates complex melodies to communicate and influence emotions |
 | Luminoth       | 0.149196 | Moth-like creature with glowing wings and antenna                    | Dense forests and jungles with bioluminescent flora | Emits soothing light patterns to communicate and attract prey  |
 | Zephyr Dancer  | 0.136456 | Graceful avian creature with iridescent feathers                     | High mountain peaks and wind-swept plains           | Creates mesmerizing aerial displays to attract mates           |
 
-Great results again! Our two other retrieved monsters - Luminoth and Zephyr Dancer - have behavior similar to Harmonic Coral, and live in habitats different from Harmonic Coral's. They also look very different from Harmonic Coral. (Harmonic's tendrils and Luminoth's antenna are similar features, but we only down-weighted `look_weight` by 0.5, and the resemblance between the two monsters ends there.)
+Great results again! Our two other retrieved monsters — Luminoth and Zephyr Dancer — have behavior similar to Harmonic Coral and live in habitats different from Harmonic Coral's. They also look very different from Harmonic Coral. (While Harmonic Coral's tendrils and Luminoth's antenna are somewhat similar features, we only down-weighted the `look_weight` by 0.5, and the resemblance between the two monsters ends there.)
+
+Here's a detailed breakdown of the scores as we did above:
+
+|                |     look |   habitat | behavior |    total |
+| :------------- | -------: | --------: | -------: | -------: |
+| Harmonic Coral |  0.19245 |   -0.3849 |   0.3849 |  0.19245 |
+| Luminoth       | 0.052457 | -0.068144 | 0.164884 | 0.149196 |
+| Zephyr Dancer  | 0.050741 | -0.079734 | 0.165449 | 0.136456 |
+
+The negative score in the `habitat` attribute indicates a down-weighting, reflecting our intention to de-emphasize `habitat` similarity in this particular query.
+
+Both Luminoth and Zephyr Dancer maintain strong `behavior` scores comparable to Harmonic Coral, indicating similar behavioral traits. Their `look` scores are positive but lower, reflecting some visual similarities while still maintaining distinct appearances. The negative `habitat` scores emphasize their different living environments compared to Harmonic Coral.
+
```

Comment:
@andrey
negative scores in `habitat` result from down-weighting, per your statement above. Seems like this is a magnitude comparison (not a negative vs positive comparison).

Reply:
You are right, we should say that we pushing similar monsters **away** and not just down-weight. Probably then like this?

The negative score for the `habitat` attribute means that monsters with similar habitat are not just down-weighted but rather "pushed away" from the Harmonic Coral during the search.